### PR TITLE
Add `ChoiceAllResponse` for `usnat` campaign

### PR DESCRIPTION
**Pull request by Nevazhnovu**

### CODE REVIEW

File: ConsentViewController/Classes/SourcePointClient/SourcepointClientCoordinator.swift [hunk 1]
```diff
@@ -766,11 +766,22 @@ class SourcepointClientCoordinator: SPClientCoordinator {
...
+        if let usnat = response.usnat, campaign == .usnat {
+            state.usnat?.dateCreated = usnat.dateCreated
+            state.usnat?.expirationDate = usnat.expirationDate
+            state.usnat?.consentStatus = usnat.consentStatus
+            state.usnat?.GPPData = usnat.GPPData
+            // state.usnat?.categories = usnat.сategories
+            // state.usnat?.consentStrings = usnat.consentStrings
+            // state.usnat?.statuses.consentedToAll = usnat.consentedToAll
+            // state.usnat?.consentStatus.rejectedAny = usnat.rejectedAny
+            // state.usnat?.statuses.gpcStatus = usnat.gpcEnabled
```
Comment (Nevazhnovu) [inline ConsentViewController/Classes/SourcePointClient/SourcepointClientCoordinator.swift:778]: Hey @andresilveirah ! 
This (commented) data is not saved for ccpa campaign, however, it is present in usnat. 
Do you think I should save it to storage?

Comment (andresilveirah) [reply ConsentViewController/Classes/SourcePointClient/SourcepointClientCoordinator.swift:778]: yes sir

File: ConsentViewController/Classes/SourcePointClient/SourcepointClientCoordinator.swift [hunk 2]
```diff
@@ -766,11 +766,22 @@ class SourcepointClientCoordinator: SPClientCoordinator {
...
+        }
         storage.spState = state
     }
 
     func shouldCallGetChoice(for action: SPAction) -> Bool {
-        (action.campaignType == .ccpa || action.campaignType == .gdpr) &&
+        (action.campaignType == .ccpa || action.campaignType == .gdpr || action.campaignType == .usnat) &&
```
Comment (andresilveirah) [inline ConsentViewController/Classes/SourcePointClient/SourcepointClientCoordinator.swift:784]: let's just remove this condition

File: ConsentViewController/Classes/SourcePointClient/SourcepointClientCoordinator.swift (hunk 1 above)
Comment (Nevazhnovu) [reply ConsentViewController/Classes/SourcePointClient/SourcepointClientCoordinator.swift]: Done!

File: ConsentViewController/Classes/SourcePointClient/SourcepointClientCoordinator.swift (hunk 2 above)
Comment (Nevazhnovu) [reply ConsentViewController/Classes/SourcePointClient/SourcepointClientCoordinator.swift]: Done!

**Pull request merged**